com.unity.render-pipelines.universal/Editor/ShaderGraph/Includes/PBRGBufferPass.hlsl

```diff
@@ -64,7 +75,8 @@ FragmentOutput frag(PackedVaryings packedInput)
     #endif
 
     InputData inputData;
-    BuildInputData(unpacked, surfaceDescription, inputData);
+    InitializeInputData(unpacked, surfaceDescription, inputData);
+    //SETUP_DEBUG_TEXTURE_DATA(inputData, unpacked.uv, _MainTex);
```

Comment:
leftover?

Reply:
I think these are left here to illustrate the problem why mip debug modes cannot be supported with ShaderGraph - we don't know what texture name to refer to (_MainTex here & builtin shaders). I'll add a comment to clarify because this is still bit of an open question.